hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -554,34 +635,32 @@ def onchange_unit_amount(self):
                 'title': _("Warning"),
```

Comment:
I would like testing a bit more to be sure about this. Remember you provided with tests 11 and 12, and both still work fine.

Reply:
I was not able to reproduce the warning anymore. I will continue to test of course but if you also cannot reproduce it I would just remove it. Your solution is obsoleting this warning, I'm quite sure about this.

Reply:
good. And why the related tests are still working?

Reply:
The tests are triggering the workaround  and checking that this workaround works. But since the workaround cannot be triggered by human interaction anymore, this code and its related tests are now obsolete and can be removed.

Reply:
ah, ok hahaha I didn't check that they were triggering specifically the workaround, so I can remove test 11, right? and the test 12?

Reply:
You could keep test 12, just updating [this line](https://github.com/OCA/hr-timesheet/blob/11.0/hr_timesheet_sheet/tests/test_hr_timesheet_sheet.py#L581) and removing [this line](https://github.com/OCA/hr-timesheet/blob/11.0/hr_timesheet_sheet/tests/test_hr_timesheet_sheet.py#L607).

Reply:
I think there is no need to update/remove those lines :S The onchange always shoot run when modifying a cell. The tests are still green :)